src/main/java/auto_ckz/common/enums/RepairStatusWrapper.java

```diff
@@ -0,0 +1,20 @@
+package auto_ckz.common.enums;
+
+
+import java.util.ArrayList;
+
+public class RepairStatusWrapper {
```

Comment:
właśnie chodziło mi o wyciągnięcie wszystkich wartości, także teraz mogę to wykorzystać :P